generators/app/templates/travis.yml

```diff
@@ -1,16 +1,19 @@
 sudo: false
 language: node_js
 cache:
+  yarn: true
   directories:
     - node_modules
 notifications:
   email: false
 node_js:
   - '6'
+before_script:
+  - yarn global add codecov
 script:
   - yarn start validate
 after_success:
-  - yarn start report-coverage
+  - codecov
   - yarn start release
```

Comment:
well, I considered to do it too but I realized semantic-release needs a one-time setup to do integration between repo, CI, and npm (github token/npm token). and that kind of integration requires user input for CLI right?